adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -121,14 +130,11 @@ class Transform(protected val args: TransformArgs) extends BDGSparkCommand[Trans
     if (args.recalibrateBaseQualities) {
       log.info("Recalibrating base qualities")
       val knownSnps: SnpTable = createKnownSnpsTable(sc)
-      val stringency = if (args.strictBQSR) {
```

Comment:
That sounds reasonable, but I got kinda lost here:

> I'd go a step further though and say that it's worth formalizing the idea that, in the context of any one `transform` command, there might be a level of strictness of sanity/validation-checking that we should expose to the user via a cmdline flag.

Just to check my understanding, are you saying that it'd be reasonable for each subpart of `transform` (e.g., BQSR, indel realignment, load from FASTQ, load from BAM, mark dups, etc) to have a validation strictness flag, but that we should only expose a single flag? If so, I think that makes sense in this case, and is probably not a bad default policy in general.